lib/views/widgets/forms/register_form.dart

```diff
@@ -74,7 +73,7 @@ class RegisterFormState extends State<RegisterForm> {
               final Token token =
                   new Token(tokenString: resultData.data['signup']['token']);
```

Comment:
Re unable to join organization, yes it was one of the issues I was working on after sunday's meeting. I also pushed the fix to it a few hours ago with relevant error and success messages. 
The current HomePage (which has the loading spinner) won't be used. It's just there from the existing code. So after merging with Dimitri the app would have begun to come together and certain existing code refactored/removed.

I will have a look at the error log...

Reply:
removing the ".data" worked. and also fixed another issue I was having where on some devices I wasn't able to get user data displayed on the navbar after registering. But it would've worked when login